### ISSUE TITLE
What is AbstractRoyalties._updateAccount() for?

### ISSUE THREAD

**Issue by php4fan** [AboldUSER/rarible-royalties-tutorial#1]
`AbstractRoyalties` has an internal function called `_updateAccount()` but it doesn't seem to be used anywhere.
If it weren't internal, I would assume that it is either used by the Rarible protocol, or meant to be called by the contract's owner "manually". But being internal and not called from anywhere it seems to me it can't be of any use. 
Am I missing something?

**Comment by AboldUSER** [comment]
You're right that it is used by the Rarible Protocol in other contracts, and it is used via inheritance of AbstractRoyalties.
If you look at Rarible's github, they inherit the RoyaltiesV2Impl contract (which inherits AbstractRoyalties) into their ERC721Lazy and ERC1155Lazy contracts, which are inherited into their base ERC721 and ERC1155 contracts respectively.
You can see in both [ERC721Lazy](https://github.com/rarible/protocol-contracts/blob/6e3b17d9fb0b0a4357139f10a130286484aace1f/tokens/contracts/erc-721/ERC721Lazy.sol#L108) and [ERC1155Lazy](https://github.com/rarible/protocol-contracts/blob/168ed48facbb8708df1d7b9eaf1fe0d73b66b4dd/tokens/contracts/erc-1155/ERC1155Lazy.sol#L130) that they use the internal _updateAccount function in the external updateAccount function.
